crates/sui-framework/packages/sui-framework/tests/random_tests.move

```diff
@@ -83,13 +650,13 @@ module sui::random_tests {
         let random_state = test_scenario::take_shared<Random>(scenario);
         update_randomness_state_for_testing(
             &mut random_state,
-            0,
+            1,
```

Comment:
why changes to the existing tests here?

Reply:
in particular for this test I think it may now fail on the first call instead of the second, which means it's not testing the same thing anymore?

Reply:
no good reason, probably debugged something...
reverted that change